### PR TITLE
Register PyPI

### PR DESCRIPTION
**Pull request by k2kobayashi**
- change installation method in README.md
- change stable ver.
- modify setup.py slightly

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,13 +10,29 @@
...
+    packages=find_packages(exclude=('docs', 'tests')),
     long_description=(README + '\n' + CHANGES),
-    license='MIT',
+
+    # requirements
     install_requires=open('requirements.txt').readlines(),
+    extras_require={
+        'develop': ['nose','docopt'],
+    },
+
```
Comment (tats-u) [inline setup.py]: `python_requires='>=3.5',`

**Comment by k2kobayashi** [comment]
Thanks @tats-u

**Comment by k2kobayashi** [comment]
After I register PyPI, I will merge and close.

**Comment by k2kobayashi** [comment]
Eventually, it works via pip. Thanks for cooperation by @tats-u .

**Pull request merged**